src/custom-hooks/useToast.jsx

```diff
@@ -0,0 +1,26 @@
+import { toast } from "react-toastify";
+
+const useToast = () => {
+	const showToast = (
+		toastText = "Task Successfull",
+		toastTheme = "success"
+	) => {
+		const notify = () => {
+			toast[toastTheme](toastText, {
+				theme: "colored",
+				position: "top-right",
+				autoClose: 1500,
+				hideProgressBar: false,
+				closeOnClick: true,
+				pauseOnHover: true,
+				draggable: true,
+				progress: undefined,
+			});
+		};
+		notify();
```

Comment:
As these are the default styles of your toast, you can pass it as a prop to the ToastContainer component itself. imo, this abstraction is redundant and also adds a limitation of using toast only inside a component or custom hook.